schema/compositor.h

```diff
@@ -22,6 +22,7 @@
 #ifndef SCHEMA_COMPOSITOR_H
 #define SCHEMA_COMPOSITOR_H
 
+#include <QArgument>
```

Comment:
I am not an expert in this field, but I would prefer including the <QMetaObject> because that has reference in the Qt documentation (see. QMetaObject), while the I have not been able to find references to the QArgument header and I am a bit concerned if it might be removed in the future.

Reply:
Changed to <QObject>